src/status_im/init/core.cljs

```diff
@@ -66,8 +66,12 @@
 (fx/defn initialize-multiaccounts
   {:events [::initialize-multiaccounts]}
   [{:keys [db] :as cofx} all-multiaccounts]
-  (let [multiaccounts (reduce (fn [acc {:keys [address] :as multiaccount}]
-                                (assoc acc address multiaccount))
+  (let [multiaccounts (reduce (fn [acc {:keys [address keycard-key-uid] :as multiaccount}]
+                                (-> acc
+                                    (assoc address multiaccount)
+                                    (assoc-in [address :keycard-key-uid] (when-not
```

Comment:
could the assoc be cond-> so that there is no :keycard-key-uid at all when keycard-key-uid is empty

Reply:
actually rereading your code it should be fixed on status-go side with omit emtpy or something

Reply:
changed to `cond->`
good if will be fixed on status-go side